doc/base/classes.xml

```diff
@@ -22141,7 +22141,7 @@
 			<argument index="0" name="id" type="int">
 			</argument>
 			<description>
-				Returns a resolved item address if one is available. Returns an empty string on error or if resolution hasn't happened yet (see [method get_resolve_item_status]).
+			git	Returns a resolved item address if one is available. Returns an empty string on error or if resolution hasn't happened yet (see [method get_resolve_item_status]).
```

Comment:
? :)

Reply:
Haha whoops, just seen that. Will fix that up too.